docs/blog-2.md

```diff
@@ -0,0 +1,209 @@
+
+# `git machete` strikes again! Traverse the git (rebase) jungle even faster with v2.0+
+
+
+## Intro
+
+Good news - git machete 2.0 has been released!
+
+For those of you not familiar with this nifty tool, I recommend having a quick look at the [previous part of this series (link)](https://virtuslab.com/blog/make-way-git-rebase-jungle-git-machete)
+or at least scrolling through the screenshots in that blog post to get a TL;DR kind of understanding of what git machete actually does.
+
+In general, git machete is a tool that helps manage multiple branches within a git repository and thus promotes the creation of smaller pull requests that are (usually) easier to review.
+Just recall the last time you had to review a giant like this (diff stats shown of course only for a few first files):
+
+![Too many files in a single PR...](files-changed.png)
+
+You can get the latest git machete release directly from [the git machete repo (github.com/VirtusLab/git-machete)](https://github.com/VirtusLab/git-machete):
+
+```
+$ git clone https://github.com/VirtusLab/git-machete.git
+$ cd git-machete
+$ sudo make install
```

Comment:
Whoops this is no longer working 😅

Reply:
In fact a single link to `Install` section in the README would be a better idea here